Dynamic-Programming/tests/UniquePaths.test.js

```diff
@@ -0,0 +1,19 @@
+import { uniquePaths } from '../UniquePaths'
+
+test('Base Case 1', () => {
```

Comment:
Please find more descriptive names for the tests or merge them.

Reply:
Can I make it into (describe - it) blocks?

Reply:
Just move everything into one `test` block for `Unique Paths DP`

Reply:
Done!